tests/builder/builder_test.py

```diff
@@ -2298,9 +2314,25 @@ def test_load_kerning_bracket(ufo_module):
         "public.kern1.foo": ["x", "x.BRACKET.wght_300_1000", "x.BRACKET.wght_600_1000"],
     }
     assert ds2.sources[0].font.groups == bracketed_groups
-    assert ds2.sources[1].font.groups == bracketed_groups
+
+    # Prior to PR #838, we were checking for d1 and d2 to be equal.
+    # However, with PR #838, we are now pruning kerning groups that are not
+    # used in the kerning table.
+    # The groups in the assertion below got pruned.
+    d1 = ds.sources[1].font.groups
+    d2 = bracketed_groups
+    assert set(d2) ^ set(d1) == {"public.kern2.foo", "public.kern1.foo"}
+
     assert ds2.sources[2].font.groups == bracketed_groups
-    assert ds2.sources[3].font.groups == bracketed_groups
+
+    # Prior to PR #838, we were checking for d1 and d2 to be equal.
+    # However, with PR #838, we are now pruning kerning groups that are not
+    # used in the kerning table.
+    # The groups in the assertion below got pruned.
```

Comment:
make this repeated comment less verbose at least, copy pasting repeatedly doesn't help